### DISCUSSION TITLE
在写uni基础组件的property时 还是没有自动补全

### DISCUSSION THREAD

**Discussion by tanpengsccd** [uni-helper/uni-app-types#42] [Q&A] [answered]
使用了这个库发现现在 在写 uni-app+vue3+ts 时能终于能看到注释和还能跳转定义了. 但是有个小问题: 
在写uni基础组件的property时 还是没有自动补全, 虽然能跳进该property 查看定义的,但是还是不是不如有自动补全方便.
比如这里
![image](https://user-images.githubusercontent.com/13959880/229591736-78d4dc7a-8f21-49a5-9658-6f73d3182385.png)

**Comment by ModyQyW** [comment, marked as answer]
我感觉这不属于这个库的问题，因为这个库只是提供 ts 类型定义，自动补全和提示应该是和 volar 相关？🤔️

PS：看到创建这个 discussion 的时间是凌晨两三点，希望这个库能帮你早点下班 ❤️

**Reply by tanpengsccd** [reply]
感谢大佬回复。我搜了下资料感觉好少。我以为大佬们有相关经验

**Reply by ModyQyW** [reply]
如果有什么资料表明这个库可以优化自动补全，可以发出来看看，目前先关闭了哈。感谢你对这个库的关注！
